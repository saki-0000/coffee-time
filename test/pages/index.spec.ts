import { shallowMount } from '@vue/test-utils'
import IndexPage from '~/pages/index.vue'

describe('CoffeeCalculator.vue', () => {
  test('renders correctly', () => {
    const wrapper = shallowMount(IndexPage)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
