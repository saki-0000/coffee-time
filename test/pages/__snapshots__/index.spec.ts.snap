// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CoffeeCalculator.vue renders correctly 1`] = `
"<v-row-stub tag=\\"div\\" align=\\"center\\" justify=\\"center\\">
  <v-col-stub cols=\\"12\\" sm=\\"8\\" md=\\"6\\" tag=\\"div\\">
    <v-card-stub loaderheight=\\"4\\" tag=\\"div\\" class=\\"logo py-4 d-flex justify-center\\">
      <coffeecalculator-stub></coffeecalculator-stub>
    </v-card-stub>
    <v-card-stub loaderheight=\\"4\\" tag=\\"div\\" class=\\"logo py-4 d-flex justify-center\\">
      <coffeetimer></coffeetimer>
    </v-card-stub>
  </v-col-stub>
</v-row-stub>"
`;
